src/resources/elements/NetworkFeedback/NetworkFeedback.ts

```diff
@@ -26,9 +26,10 @@ export class NetworkFeedback {
     const isTestnet = [Networks.Rinkeby, Networks.Alfajores, Networks.Kovan].includes(storedNetwork);
     if (
       storedNetwork &&
-      (this.isProduction && !isMainnet) || (!this.isProduction && !isTestnet)
+      (this.isProduction && isTestnet) || (!this.isProduction && isMainnet)
```

Comment:
1.How can `this.isProduction && isTestnet` ever be true?

2. Can we have more descriptives if-statements in the future?
```ts
const A = this.isProduction && isTestnet
const B = !this.isProduction && isMainnet
const C = A || B
const D = storedNetwork && C
```